lib/widgets/message_list.dart

```diff
@@ -1251,14 +1241,31 @@ class MessageWithPossibleSender extends StatelessWidget {
           Text(time,
             style: TextStyle(
               color: messageListTheme.messageTimestamp,
-              fontFamily: 'Source Sans 3',
               fontSize: 16,
               height: (18 / 16),
               fontFeatures: const [FontFeature.enable('c2sc'), FontFeature.enable('smcp')],
             ).merge(weightVariableTextStyle(context))),
         ]);
     }
 
+    final localizations = ZulipLocalizations.of(context);
+    String? editStateText;
+    switch (message.editState) {
+      case MessageEditState.edited:
+        editStateText = localizations.messageIsEditedLabel;
+      case MessageEditState.moved:
+        editStateText = localizations.messageIsMovedLabel;
+      case MessageEditState.none:
+    }
+
+    final editStateLabel = editStateText == null ? null : Text(editStateText,
+      textAlign: TextAlign.end,
```

Comment:
nit:
```suggestion
    final editStateLabel = editStateText == null ? null
      : Text(editStateText,
          textAlign: TextAlign.end,
```

That way the `Text` constructor name, and `editStateText` which holds the actual text content, stay more visible and next to the other arguments to `Text`.

Reply:
Start `textAlign` in a new line for the now inlined `editStateLabel` because we don't need to render another copy now.